src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+public class Ball {
+    private String color;
+    private int number;
+    public void setColor(String color) {
+        this.color = color;
+    }
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+    public Ball() {
+    }
+    public void setNumber(int number) {
+        this.number = number;
+    }
```

Comment:
Similarly to the setColor method, consider if the setNumber method is necessary. If the number is not supposed to change after the object is created, you can remove this setter.